_posts/en/newsletters/2020-09-30-newsletter.md

```diff
@@ -8,7 +8,9 @@ layout: newsletter
 lang: en
 ---
 This week's newsletter describes a compiler bug that casts doubt on the
-safety of secure systems and includes our regular sections with popular
+safety of secure systems and explains a technique that can be used to more
+efficiently verify ECDSA signatures. Also included are our regular sections
```

Comment:
Maybe s/ECDSA signatures/ECDSA signatures in Bitcoin/ (as, IIUC, endomorphism only applies to certain curves.

Reply:
done